dashboard/src/views/SiteOverview.vue

```diff
@@ -86,8 +92,10 @@ const marketplacePromotionalBanners = useResource({
 				</template>
 			</Alert>
 		</div>
-
-		<Alert title="Trial" v-if="isInTrial && $account.needsCard">
+		<Alert title="Trial" v-if="isTrialEnded">
+			Your trial has ended. Select a plan from the Plan section below to continue using your site.
+		</Alert>
+		<Alert title="Trial" v-if="isInTrial && $account.needsCard && !isTrialEnded">
```

Comment:
```suggestion
		<Alert title="Trial" v-if="isInTrial && $account.needsCard">
```

`isInTrial` must be `false` when `hasTrialEnded` is `true`, single source of truth.

Reply:
added validation to make isInTrial as single source of truth